build/ProjectReunion-Create-VSIX.yml

```diff
@@ -0,0 +1,96 @@
+parameters:
+  - name: "ReunionVersion"
+    type: string
+    default: ''
+  - name: "ReunionFoundationVersion"
+    type: string
+    default: ''
+  - name: "ReunionDWriteVersion"
+    type: string
+    default: ''
+  - name: "ReunionWinUIVersion"
+    type: string
+    default: ''
+  - name: "releaseBuild"
+    type: boolean
+    default: True
+
+jobs:
+
+- job: CreatingVsix
+  pool:
+    vmImage: 'windows-2019'
+
+  variables:
+    ${{if eq(parameters.releaseBuild, True) }}:
+      VSIXBuildArgs: '/p:"RestoreSources=https://api.nuget.org/v3/index.json;https://pkgs.dev.azure.com/microsoft/ProjectReunion/_packaging/Project.Reunion.Internal.ReleaseSigned/nuget/v3/index.json"'
+    ${{if eq(parameters.releaseBuild, False) }}:
+      VSIXBuildArgs: ''
+      
+  steps:
+  - task: NuGetAuthenticate@0
+    inputs:
+      nuGetServiceConnections: Internal-ReleaseSigned
+
+  # NuGetCommand@2 
```

Comment:
I don't think this is necessary, but it would be given the VSToolsPath override below.  Can we remove all of this and try another build?  Local builds don't actually require the VSSDK nuget package, but use the VS built-in tools path.